app/cdap/components/common/InlayDrawerWidget/Menu.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
```

Comment:
wrong copyright year and remove common folder

Reply:
updated the copyright year to 2022